e2e/fixtures/register.test.js

```diff
@@ -0,0 +1,22 @@
+import { URLS } from '../utils/route.util';
+import { RegisterElements } from '../page-objects/register.po';
+//Declare chance
+const Chance = require('chance');
+const chanceObj = new Chance();
+
+const User = chanceObj.name();
+const Username = User.toLowerCase();
```

Comment:
make your variables lowercase
```suggestion
const username = user.toLowerCase();
```